### DISCUSSION TITLE
Strangeness with Sonic on an NTSC Breadbin

### DISCUSSION THREAD

**Discussion by GorrillaRibs** [frntc/RAD#36] [Q&A] [answered]
Hi, to preface I'm using a RAD with a Raspi 3, and in general it seems to be working just fine with the default settings (GEOS recognizes it and I can use it as a RAM drive), but when playing the Sonic the Hedgehog port there's some graphical flickering when the screen scrolls (the video linked below) - there doesn't seem to be any other issues playing the game (aside from the graphical glitches, scrolling is smooth, music plays properly, etc), so I'm unsure how to go about debugging the issue. Any advice? Is this a known quirk of the RAD? Any help would be appreciated!

(Also the RAD is pretty rad, thanks for the project!)

Video: [https://youtube.com/shorts/1R5I1wJ094U?feature=share](https://youtube.com/shorts/1R5I1wJ094U?feature=share)

**Comment by hpingel** [comment, marked as answer]
An NTSC Commodore 64 is too slow for Sonic The Hedgehog. You need a C128 NTSC or something else with a faster CPU.

This is described in the documentation of the Sonic release. Graphical glitches on C64 NTSC are to be expected. This is not related to RAD Expansion Unit and will be similar with a real REU.

**Reply by GorrillaRibs** [reply]
Good to know then! I tested in VICE emulating an NTSC 64 and didn't get the flickering, but it could be VICE is just smoothing over some issue on real hardware.
